src/Player.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by strejolopez on 11/29/16.
+ */
+public class Player {
+    private String name;
+    private String token;
+    private int money;
+
+    public Player(String name, String token) {
+        this.name = name;
+        this.token = token;
+        this.money = 1500;
+    }
+
+    public String getName() { return this.name; }
+    public String getToken() { return this.token; }
+    public int getMoney() { return this.money; }
+
+
+    public void payMoney(int amount){
+        if(money < 1500) {
+            this.money -= amount;
+        System.out.println();
+    }
+    else{
+            System.out.println(this.money-= amount);
+            System.out.println("Losing money");
+        }
+        }
+
+    public void receiveMoney(int amount){
+        if(money > 1500) {
```

Comment:
Avoid hardcoding numbers like 1500.  In this instance if you get paid 5000 then 1500 as a limit makes no sense.